apps/chat/src/components/Common/ModelsSelector.tsx

```diff
@@ -61,25 +61,44 @@ interface ModelsSelectorProps {
   disabled?: boolean;
   tooltip?: string;
   onChange: (modelId: string) => void;
+  models?: DialAIEntityModel[];
+  additionalModelsMap?: Partial<Record<string, DialAIEntityModel>>;
+  useReference?: boolean;
+  inputClassName?: string;
+  panelClassName?: string;
+  indexSeparator?: number;
 }
 
 export const ModelsSelector = memo(function ModelsSelector({
   value,
   disabled,
   tooltip,
   onChange,
+  models,
+  additionalModelsMap,
+  useReference,
+  inputClassName,
+  panelClassName,
+  indexSeparator,
 }: ModelsSelectorProps) {
   const onlyModels = useAppSelector(ModelsSelectors.selectModelsOnly);
```

Comment:
```suggestion
  const modelTypeAgents = useAppSelector(ModelsSelectors.selectModelsOnly);
```

sounds like a boolean
lets rename it? ex. `modelTypeAgents`